.travis.yml

```diff
@@ -13,9 +13,17 @@ addons:
 script:
     - shellcheck *.sh
     - ./test-build.sh $NODE_VERSION
+    - if [ "true" = "${DOCTOCCHECK-}" ]; then
```

Comment:
Should this be above `./test-build.sh $NODE_VERSION`? Would suck to spend half an hour waiting for the build only for the TOC to be out of date